docs/sources/mimir/references/architecture/deployment-modes/index.md

```diff
@@ -11,27 +11,39 @@ weight: 20
 
 # Grafana Mimir deployment modes
 
-You can deploy Grafana Mimir in three modes:
+Grafana Mimir offers two deployment modes to accommodate different operational requirements and scale needs. Choose the deployment mode that best fits your use case:
 
-- Monolithic mode
-- Microservices mode
-- Read-Write mode (experimental)
+- Monolithic mode: Run all components in a single process for simple deployments.
+- Microservices mode: Deploy components separately for maximum scalability and flexibility.
 
-The deployment mode is determined by the `-target` parameter, which you can set via CLI flag or YAML configuration.
+Configure the deployment mode using the `-target` parameter, which you can set via CLI flag or YAML configuration.
 
-## Monolithic mode
+## About monolithic mode
 
-The monolithic mode runs all required components in a single process and is the default mode of operation, which you can set by specifying `-target=all`. Monolithic mode is the simplest way to deploy Grafana Mimir and is useful if you want to get started quickly or want to work with Grafana Mimir in a development environment. To see the list of components that run when `-target` is set to `all`, run Grafana Mimir with the `-modules` flag:
+Monolithic mode runs all required components in a single process and is the default deployment mode. You enable monolithic mode by setting `-target=all`.
+
+This mode is ideal when you want to:
+
+- Get started quickly with Mimir
+- Run Mimir in a development environment
+- Deploy with minimal operational complexity
```

Comment:
This makes microservices mode sound like it's an optimization when it's actually the right way to be running Mimir. I would remove this and the next point. We don't want people to try running monolithic mode anywhere outside of "getting started".